joomla/joomla-cms
# [5.1] Convert mod_multilangstatus to service provider

### Summary of Changes

Converts the multilanguage status module to service provider.

### Testing Instructions

1. Make sure you have administrator module mod_multilangstatus and System - Language Filter plugin published
2. You should see module output in administrator

![mod_multilangstatus](https://github.com/joomla/joomla-cms/assets/59166858/1b631af1-5baf-43e8-bffe-794d4279df43)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
